data/countries/SE.yaml

```diff
@@ -16,6 +16,7 @@ holidays:
       01-05:
         name:
           sv: Trettondagsafton
+        type: observance
```

Comment:
Shouldn't the "De facto half holidays" be better attributed a "optional" as it looks to me that most people take the half day off.
To mark the half days wouldn't it be good to correct them to `01-05 12:00:`.

Reply:
Thanks for the feedback!

I have updated these holidays to "optional" per your suggestion. According to my colleagues it is increasingly rare that people take time off during these holidays, but the stock market does close after 12:00.